plone_training_config/manifests/plone.pp

```diff
@@ -1,28 +1,28 @@
 class plone {
 
-    $plone_version = "5.0.6"
+    $plone_version = "5.0.7"
```

Comment:
I'd prefer to use 5.1rc1 for the training. Also 5.0.8 is already out and 5.0.9 is pending.

Reply:
The `plone_version` in the manifest is only used for the buildout cache download. Imo it should match the version in https://github.com/collective/training_buildout. So what we need to change is the training buildout which currently uses 5.0.7.